packages/components/src/index.ts

```diff
@@ -1 +1,3 @@
+export * from './ErrorBoundary';
 export { Interactive } from './Interactive';
```

Comment:
Why isn't everything from Interactive exported like other modules? When I tried that, I got a bunch of type errors. I didn't spend much time on it, but I think it had to do with Clickable's props type depending on Interactive's. Left it for another day.

Reply:
I have no problem with this.... In fact I kinda cringe with how much we use `import * from 'path/some-module'`